src/services/services.ts

```diff
@@ -6052,7 +6082,11 @@ namespace ts {
                 return getReferencesForSuperKeyword(node);
             }
 
-            const symbol = typeChecker.getSymbolAtLocation(node);
+            const isConstructor = node.kind === SyntaxKind.ConstructorKeyword;
+
+            // `getSymbolAtLocation` normally returns the symbol of the class when given the constructor keyword,
+            // so we have to specify that we want the constructor symbol.
+            let symbol = isConstructor ? node.parent.symbol : typeChecker.getSymbolAtLocation(node);

```

Comment:
Never mind, that doesn't make sense because node.parent presumably also returns the class symbol.